server/handlers/error.go

```diff
@@ -207,9 +207,8 @@ func ErrExecutionPlan(err error) error {
 }
 
 func ErrCompConfigPairs(err error) error {
-	return errors.New(ErrCompConfigPairsCode, errors.Alert, []string{"unable to Create Comp Config.", err.Error()}, []string{err.Error()}, []string{}, []string{})
+	return errors.New(ErrCompConfigPairsCode, errors.Alert, []string{"Unable to Create Component Configuration.", err.Error()}, []string{("An error occurred while trying to create Component Configuration."), err.Error()}, []string{"Missing or invalid user token, or an issue with Meshery's interactions with Kubernetes."}, []string{"For understanding and troubleshooting, refer to https://docs.meshery.io/guides/troubleshooting and https://docs.meshery.io/installation/kubernetes"})
```

Comment:
Add name of the component for which it failed, (its kind and name).
> "Missing or invalid user token, or an issue with Meshery's interactions with Kubernetes." 
only single cause?
1. No registered model/comp  found inside registry
2. failed validation

Will you verify what error you get if we try to deploy a CR inside a cluster which doesn't have a CRD deployed.

What error you get from meshery vs when you do same from kubectl